app/views/devise/shared/_password_strength.html.erb

```diff
@@ -10,9 +10,9 @@
       <span class='h6'>
         <%= t('instructions.password.strength.intro') %>
       </span>
-      <span id='pw-strength-txt' class='bold' data-forbidden-passwords='<%= @forbidden_passwords %>'>
-        ...
-      </span>
+      <%= tag.span '...', id: 'pw-strength-txt', class: 'bold', data: {
+        'forbidden-passwords': @forbidden_passwords,
+      } %>
```

Comment:
is it worth adding a view spec for this partial as a regression spec to make sure this continues to be encoded correctly?

Reply:
Also WDYT of changing the attribute name to just `forbidden` so that way we don't have to string-quote the key and have to do less camelcase translation in JS?

Reply:
> is it worth adding a view spec for this partial as a regression spec to make sure this continues to be encoded correctly?

Added in 7d1215a8d992e065c68cafdd90dda08bfa6faad2.

Reply:
> Also WDYT of changing the attribute name to just `forbidden` so that way we don't have to string-quote the key and have to do less camelcase translation in JS?

Updated in cbe6787.